graylog2-web-interface/src/stores/nodes/NodesStore.js

```diff
@@ -43,6 +43,14 @@ const NodesStore = Reflux.createStore({
     return this.nodes[nodeId];
   },
 
+  getClusterId() {
+    return Object.keys(this.nodes).map(id => this.nodes[id]).map(node => node.cluster_id)[0].toUpperCase();
```

Comment:
Why not doing something like `.find(clusterId => clusterId)` instead of `[0]`, so we protect against possible undefined/null cluster ids?